client/index.html

```diff
@@ -59,23 +59,32 @@
       <span id='close-button'><div class="icon-angle-left"></div></span>
       <div class="tree_info">
         <img id="tree-image" src="">
-	<div class="planter_image">
-		<img id="planter_image"t sta src="img/portrait_placeholder_100.png">
-	</div>
-	<div class="planted_by">
-    <p><span>Planted by: </span><span id="planter_name">Planter</span></p>
-	</div>
+        <div class="planter_image">
+          <img id="planter_image"t sta src="img/portrait_placeholder_100.png">
+        </div>
+        <div class="planted_by">
+          <p><span>Planted by: </span><span id="planter_name">Planter</span></p>
+        </div>
+
         <div id="created_on" class="info_data">
           <p>Created on : </p>
           <p id="create-data"></p>
         </div>
-
         <div id="status" class="info_data">
           <p>Status : </p>
           <p id="status-data">Planted</p>
         </div>
-	</div>
-        
+
+        <br>
+        <div class="float-left info_data">
+            <a href="#" class="float-left tree_nav" id="tree_prev" >
```

Comment:
Should these `floats` be in the css?

Reply:
These are classes that were already in the style.css file. Not sure the best practices for css though.